### DISCUSSION TITLE
[UML] 시퀀스 다이어그램 (Sequence Diagram) 란 무엇인가

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#21] [Q&A] [answered]
Sequence Diagram

**Comment by tjdud22** [comment, marked as answer]
시퀀스 다이어그램(Sequence Diagram)은 어떠한 순서로 어떤 객체들과 어떻게 상호작용했는지를 표현하는 다이어그램입니다.이 다이어그램은 현재 존재하는 시스템이 이떠한 시나리오로 움직이고 있는지를 나타내는데 장점을 가지고 있습니다.
